### PR TITLE
Create Animals Table

### PR DESCRIPTION
**Pull request by sunga12**
1. Created a database named vet_clinic.
2. Created a table named animals with the given columns:
3. Inserted the given data
4. Wrote queries for the following quests:

- animals whose name ends in "mon".
- name of all animals born between 2016 and 2019.
- name of all animals that are neutered and have less than 3 escape attempts.
- date of birth of all animals named either "Agumon" or "Pikachu".
- name and escape attempts of animals that weigh more than 10.5kg
- animals that are neutered.
- animals not named Gabumon.
- animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

### CODE REVIEW

**Comment by sunga12** [comment]
Here are the screenshots of the queries:

animal table screenshot 1

animal table screenshot 2

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(
+	id 	INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY NOT NULL,
+	name VARCHAR(25) NOT NULL,
+	date_of_birth DATE,
+	escape_attempts INT,
+	neutered BOOL,
```
Comment (addod19) [inline schema.sql]: - [ ] It would be better if we replace `BOOL` with `boolean` instead for the data type

**Review by addod19 — CHANGES_REQUESTED** [review]
## Hi @sunga12, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Correct Git-flow followed :100: 
- [ ] Schema has the right columns :rocket: 
- [ ] Screenshots of the query results present in the PR :rocket: 


## Required Changes ♻️
_Check the comments under the review._
- [ ] Although you have used the correct README template and which contains all the relevant headers, we are required to customize it according to the current project. Kindly take your time and edit all the sections or headers of the README file before submitting another code review request

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi @sunga12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Well-implemented code 👍 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — COMMENTED** [review]
## Hi @sunga12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] All requirements has been met :rocket: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**